bodhi/server/scripts/check_policies.py

```diff
@@ -48,6 +48,11 @@ def check():
     ).filter(
         models.Release.state.in_(
             [models.ReleaseState.current, models.ReleaseState.pending])
+    ).filter(
+        models.Update.test_gating_status.notin_(
+            models.TestGatingStatus.passed,
+            models.TestGatingStatus.ignored
+        )
```

Comment:
We don't want to do this - this is this way on purpose because Greenwave can change its mind later, after having passed. We actually had a meeting about this and chose to go this way. This proposed change takes the "Continuous" out of "Continuous Integration".

Reply:
I've discussed this with the folks running the CI pipeline and they agreed that once something has passed it should not be reconsidered, which of course isn't true for failures.

I've pinged them so they can join the discussion here.